recipes/typing-inspect/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "typing_inspect" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cf41eb276cc8955a45e03c15cd1efa6c181a8775a38ff0bfda99d28af97bcda3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - typing >=3.6.1
+    - mypy_extensions >=0.3.0
+
+test:
+  imports:
+    - typing_inspect
+
+about:
+  home: http://github.com/ilevkivskyi/typing_inspect
+  license: MIT
+  license_family: MIT
```

The licence file isn't packaged (which is a requirement of the MIT licence). If it's included in the tarball you can specify it's name like this:
```suggestion
  license_file: LICENCE
```
Else you can copy the licence from the upstream repository and include it in the recipe directory (and then specify it's name like above).

---

Other than that it looks good to go 👍

---

The file is not present either in the tarball or in the upstream repository. Is it reasonable for me to add the default MIT license file with the package author's name and current year?

---

That seems reasonable to me, though the year should probably be `2017-2019` i.e. the first commit.

Can you also make a PR upstream to add a copy of the licence and include it in the `MANIFEST.in` file?

---

Yes, was going to do that as well. It is here already.

---

Upstream PR: https://github.com/ilevkivskyi/typing_inspect/pull/33